### PR TITLE
List pages column changes

### PR DESCRIPTION
**Pull request by AvihayMReblaze**
…dded

### CODE REVIEW

File: src/views/documentListConst.ts [hunk 1]
```diff
@@ -295,6 +344,13 @@ export const COLUMN_OPTIONS_MAP: ColumnOptionsMap = {
     },
   ],
   'contentfilterprofiles': [
+    {
+      title: 'ID',
+      fieldNames: ['id'],
+      isSortable: true,
+      isSearchable: true,
+      classes: 'width-130px',
```
Comment (Aviv-Galmidi) [inline src/views/documentListConst.ts:352]: Can this be 120px like all the others? If so, then we won't need a new class for `width-130px`

Comment (AvihayMReblaze) [reply src/views/documentListConst.ts:352]: There was a long ID then 120px can contain, so i had create the 130px

Comment (Aviv-Galmidi) [reply src/views/documentListConst.ts:352]: So let's have all tables use the `width-130px` class to keep it consistent across the board

Comment (AvihayMReblaze) [reply src/views/documentListConst.ts:352]: Done

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**